frontend/pages/beta/model/new.tsx

```diff
@@ -39,6 +40,7 @@ export default function NewModel() {
 
   async function onSubmit(event) {
     event.preventDefault()
+    setLoading(true)
```

Comment:
You never run `setLoading(false)`!  If the 'createModel' request fails (e.g. with a missing field) you'll never be able to resubmit it.